schema/testing.proto

```diff
@@ -180,7 +106,7 @@ message RegisterTestRequest {
 message Session {
   // The name of the session. The ID must conform to ^[a-z]+$
   // If this is not provided, Showcase chooses one at random.
-  string name = 1 [(google.api.resource).path = "sessions/*"];
+  string name = 1 [(google.api.resource).path = "sessions/{session}"];
 
   // The specification versions understood by Showcase.
   enum Version {
```

Comment:
Document what the user gets if sending `VERSION_UNSPECIFIED` (I assume it will be the latest.)